modules/oit/include/modules/oit/processors/rasterizer.h

```diff
@@ -81,6 +89,7 @@ class IVW_MODULE_OIT_API Rasterizer : public Processor {
 
 private:
     RasterizeHandle handle_;
+    std::shared_ptr<Rasterization> rasterization_;
```

Comment:
why?

Reply:
For setting the volume ID uniforms the renderer needs to match each rasterizer with the correct ID.

https://github.com/inviwo/inviwo/blob/5afed4df451df4a678e6448a811cb148c22c56b6/modules/oit/src/processors/rasterizer.cpp#L59-L61

https://github.com/inviwo/inviwo/blob/5afed4df451df4a678e6448a811cb148c22c56b6/modules/oit/src/processors/meshvolumerenderer.cpp#L223-L233

Reply:
ahh ic. Would it not be easier to use the Rasterizer processor for id instead?

Reply:
made changes to use the identifier instead.

Reply:
Remove the ptr?